src/css/bookstore.css

```diff
@@ -279,6 +279,10 @@ body {
   opacity: 1 !important;
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230429723-a1cd1fb7-578a-41c7-9b65-9856102b1f7a.png)

- [ ] Kindly add more details in your pull request summary to make your pull request professional and descriptive enough 👍